decidim-core/spec/tasks/decidim_tasks_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "support/tasks"
+
+# rubocop:disable RSpec/DescribeClass
+describe "decidim:right_to_be_forgotten", type: :task do
```

Comment:
In `decidim-core/spec/support/tasks.rb` you add docs saying:

> Favor including "rake "-prefix as in the 2nd example above as it produces
> doc output that makes it clear a rake task is under test and how it is
> invoked.

but you're not adding the "rake" part here, can you fix it please?